dissect/target/loaders/mqtt.py

```diff
@@ -341,9 +341,20 @@ def _on_id(self, hostname: str, payload: bytes) -> None:
             self.mqtt_client.subscribe(f"{self.case}/{host}/DISKS")
             self.mqtt_client.subscribe(f"{self.case}/{host}/READ/#")
             if self.command is not None:
+                self.mqtt_client.subscribe(f"{self.case}/{host}/CALLID")
                 self.mqtt_client.publish(f"{self.case}/{host}/COMM", self.command.encode("utf-8"))
             time.sleep(1)
 
+    def _on_callId(self, hostname: str, payload: bytes) -> None:
```

Comment:
```suggestion
    def _on_call_id(self, hostname: str, payload: bytes) -> None:
```
please use snake_case